src/modules/storage/LedgerStorage.ts

```diff
@@ -579,10 +579,11 @@ export class LedgerStorage extends Storages
     }
     /**
      * Get validators
-     * @param height: The height parameter is optional.
+     * @param height The height parameter is optional.
```

Comment:
A better way to explain it:
```suggestion
     * @param height If present, the height at which the returned list of validators will apply.
                       If absent, the highest height this stoa instance is aware of is assumed.
```